### PR TITLE
add color for success/nothing changed messages

### PR DESCRIPTION
**Pull request by Alizter**
Makes it a bit easier to see what is happening.

### CODE REVIEW

**Review by JasonGross — COMMENTED** [review]
Do you think it's worth adding another argument to the log functions for colors, and only printing colors to pipes that are ttys or something?  (And/or having a --no-colors flag?)

**Comment by Alizter** [comment]
Hmm I also wondered the same. Let me see what I can do.

**Comment by Alizter** [comment]
I added a `--no-color` option accessible through kwargs. There is now a util.color function that takes a string and a color and outputs the colored string. I also wanted to see if the -color option to coqc could also be controlled but I couldn't work out where that is passed.

**Comment by JasonGross** [comment]
> I also wanted to see if the -color option to coqc could also be controlled but I couldn't work out where that is passed.

What do you want to do with this option?

File: find-bug.py [hunk 1]
```diff
@@ -178,6 +178,8 @@
                     help='If given, ensure that the computed error message occurs in this log.')
 parser.add_argument('-y', '--yes', '--assume-yes', dest='yes', action='store_true',
                     help='Automatic yes to prompts. Assume "yes" as answer to all prompts and run non-interactively.')
+parser.add_argument('--no-color', dest='coloron', action='store_const', const=False, default=True,
```
Comment (JasonGross) [inline find-bug.py:181]: Slight preference for
```suggestion
parser.add_argument('--no-color', dest='color_on', action='store_const', const=False, default=True,
```
(here and elsewhere)

Comment (Alizter) [reply find-bug.py]: Done

**Comment by Alizter** [comment]
I'm happy to merge if you are.

**Comment by JasonGross** [comment]
Let's merge if CI passes

**Pull request merged**